tools/pfe-tools/custom-elements-manifest.ts

```diff
@@ -30,6 +32,8 @@ export function pfeCustomElementsManifestConfig(options?: Config): Config {
       deprecatedDescriptionInlineTagPlugin(),
       dedentDescriptionsPlugin(),
       summaryPlugin(),
+      ecmaPrivateClassMembersPlugin(),
```

Comment:
Does this basically allow us to create any function we want private inside of a BaseClass with the prefix '#'? Just double checking I'm reading what this is doing correctly.